test/libsolidity/syntaxTests/errors/usage_standalone_no_call.sol

```
@@ -0,0 +1,7 @@
+error E();
+function f() pure {
+    // TODO is it a problem that we do not get an error here?
+    // we should at least get "statement has no effect"
+    E;
+}
+// ----
```

Not a problem in my opinion but this indicates that maybe there could be some constructs that should return an error but don't. How about these?

```solidity
(E);
(E, E, E);
uint a = E;
uint b = (E, 2);
function () f = E;
E + E;
E.selector;
type(E);
new E;
E[1];
return E;
E e;
```

---

We should set it to pure.